schema/alpha/artifacts/executable.json

```diff
@@ -1,6 +1,31 @@
 {
     "$id": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/executable.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "examples": [{
+        "category": "artifact",
+        "id": "1234",
+        "input_type": [
+            {
+                "apiversion": "v1",
+                "category": "artifact",
+                "kind": "data_set",
+                "namespace": "my-company.org"
+            }
+        ],
+        "kind": "my-traning-code",
+        "namespace": "my-company.org",
+        "output_type": [
+            {
+                "apiversion": "v1",
+                "category": "artifact",
+                "kind": "model",
+                "namespace": "my-company.org"
```

Comment:
I would assume namespace is optional

Reply:
Yes, `output_type` now is unconstrained as it is defined in the following section as 
```
"output_type": {
                    "items": {
                        "type": "object"
                    },
                    "type": "array"
                },
```